v2/bigquery-to-parquet/src/main/java/com/google/cloud/teleport/v2/templates/BigQueryToParquet.java

```diff
@@ -168,10 +168,7 @@ public interface BigQueryToParquetOptions extends PipelineOptions {
         optional = true,
         description = "Maximum output shards",
         helpText =
-            "The maximum number of output shards produced when writing. A higher number of shards"
-                + " means higher throughput for writing to Cloud Storage, but potentially higher"
-                + " data aggregation cost across shards when processing output Cloud Storage"
-                + " files.")
+            "(Optional) The number of output file shards. The default value is 1.")
```

Comment:
I think we can delete "(Optional)" from these, because they will appear under "Optional parameters" heading in the readme and docs.

Reply:
resolved in https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1490/commits/948545e5048b64dfc12e2a2db36675f11f580a17